README.md

```diff
@@ -133,9 +101,16 @@ The call to `appmetrics.monitor()` starts the data collection agent, making the
 
 You should start your application using the `node` command as usual (**not** `node-hc`).
 
-You must call `require('appmetrics');` *before* the require statements for any npm modules you want to monitor. Appmetrics must be initialized first so that it can instrument modules for monitoring as they are loaded. If this is a problem due to the structure of your application you can require the module on the node command line with -r to make sure it is pre-loaded:
+You must call `require('appmetrics');` *before* the require statements for any npm modules you want to monitor. Appmetrics must be initialized first so that it can instrument modules for monitoring as they are loaded. If this is a problem due to the structure of your application you can require the module on the node command line with -r or --require to make sure it is pre-loaded:
```

Comment:
I'd also mention `export NODE_OPTIONS="require appmetrics/start"`, as that works with docker images and `npm start` (which node-hc does not).

Reply:
@gibfahn I've updated the README to also mention NODE_OPTIONS and I've changed 'deprecated' to 'removed' as I think deprecated usually means the function is still available.